modules/ROOT/examples/live-demos/comments-callback/example.js

```diff
@@ -169,15 +169,40 @@ const tinycomments_lookup = ({ conversationUid }, done, fail) => {
     });
 };
 
+const tinycomments_fetch = (conversationUids, done, fail) => {
+  fetch(`https://api.example/conversations`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ uids: conversationUids }),
+  })
+    // .then(response => {
+    //   if (!response.ok) {
+    //     throw new Error('Network response was not ok');
+    //   }
+    //   return response.json();
+    // })
```

Comment:
Do we want to show commented code?

Reply:
@tiny-ben-tran I marked this as commented code so we could discuss. Do we want to provide example in demo for error handling if the fetch fails to return the conversations?

Reply:
Errors are already handled by `catch` below. I don't think the commented code is relevant anymore

Reply:
```suggestion
```